### PR TITLE
Add ability to drag and drop sounds

### PR DESCRIPTION
**Pull request by woofers**
Adds a handle to drag and drop any sample into other external applications or file viewers.

https://i.imgur.com/jAr6bON.mp4

### CODE REVIEW

File: renderer/components/sample.tsx [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 import Stack from './stack';
 import getFileName from '../../util/getFileName';
 
 const Sample = ({ sound }: { sound: Sound }) => {
     const theme = useTheme();
+    const ipcService = useIpcService();
     const { filename: path } = sound;
     const filename = getFileName(path);
+    const onDrag = (e) => {
+        event.preventDefault();
```
Comment (julesyoungberg) [inline renderer/components/sample.tsx]: `event` should be undefined.
```suggestion
        e.preventDefault();
```

**Review by julesyoungberg — APPROVED** [review]
found something small but otherwise looks good :+1:

File: renderer/components/sample.tsx (hunk 1 above)
Comment (woofers) [reply renderer/components/sample.tsx]: @julesyoungberg Good find, weird that it did not show any errors.

Comment (julesyoungberg) [reply renderer/components/sample.tsx]: yeah, but that's javascript for ya 🤷

**Pull request merged**